src/canvas-view/examples/canvas.js

```diff
@@ -1,6 +1,6 @@
 /**
  * @ngdoc directive
- * @name patternfly.canvas.component:pfCanvas
+ * @name patternfly.canvas.directive:pfCanvas
```

Comment:
Hi, why the switch from `component` to `directive`?

Reply:
Needed for grunt-ui-docs to even show the component on the generated doc webpage.

Reply:
Ok, yeah, I thought it would be something like that. ok